src/content/structured/community/roadmap.mdx

```diff
@@ -3,76 +3,69 @@ path: "/community/roadmap"
 
 navPriority: 1
 
-date: "2023-04-03"
+date: "2023-04-13"
 
 title: "Roadmap"
 
-subTitle: "Our roadmap of recently released work and the work we have planned."
+subTitle: "Our roadmap shows the latest work and the work we have planned."
 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/community/roadmap.mdx"
 ---
 
 ## Introduction
 
-The roadmap shows what's new, what we're working on and our future plans.
-
 The purpose of the roadmap is to help teams understand when features are likely to be delivered. Some items might change due to priorities or organisation strategies.
 
 See our [GitHub](https://github.com/orgs/mi6/projects/2/views/1) for more details and our day-to-day activities.
 
 ## What's new
 
-We recently added:
```

Comment:
I think it makes more sense to keep the original line here instead of the new line

Reply:
I don't think this original line makes sense with the rest of the wording of this section now

Reply:
Perhaps "we recently" instead then